docs/pages/versions/unversioned/sdk/audio.md

```diff
@@ -232,6 +232,11 @@ The rest of the API for `Audio.Sound` is the same as the imperative playback API
 
 ## Recording sounds
 
+> **Notes on web usage:**
+>
+> - A MediaRecorder issue on Chrome produces WebM files missing the duration metadata. [See the open Chromium issue](https://bugs.chromium.org/p/chromium/issues/detail?id=642012)
+> - MediaRecorder encoding options and other configurations are inconsistent across browsers, utilising a Polyfill such as [kbumsik/opus-media-recorder](https://github.com/kbumsik/opus-media-recorder) or [ai/audio-recorder-polyfill](https://github.com/ai/audio-recorder-polyfill) in your application will improve your experience. Any options passed to `prepareToRecordAsync` will be passed directly to the MediaRecorder API and as such the polyfill.
```

Comment:
Is it worth mentioning that this will only work in secure contexts and not plain http? Perhaps even the `No media devices available` message could mention this if we can be sure that is the cause.

Perhaps:

```suggestion
> - MediaRecorder encoding options and other configurations are inconsistent across browsers, utilising a Polyfill such as [kbumsik/opus-media-recorder](https://github.com/kbumsik/opus-media-recorder) or [ai/audio-recorder-polyfill](https://github.com/ai/audio-recorder-polyfill) in your application will improve your experience. Any options passed to `prepareToRecordAsync` will be passed directly to the MediaRecorder API and as such the polyfill.
> - Web browsers require sites to be served securely in order for them to listen to a mic. See  [MediaDevices#getUserMedia Security](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#security) for more details.
```


Thanks for this addition btw, I'm using it locally from [https://github.com/Jamedjo/expo/tree/jej/include-web-recording@expo-av](https://github.com/Jamedjo/expo/tree/jej/include-web-recording%40expo-av) until it merges.

Reply:
Thanks @Jamedjo I think it's a useful addition, it's all a little complicated to use with different rules and support across browsers